lib/screens/home/explorer/proposals_flow/proposals_details/proposal_details_screen.dart

```diff
@@ -260,11 +260,10 @@ class _ProposalDetailsScreenState extends State<ProposalDetailsScreen> {
                   abstain: _proposal.abstainAmount,
                   total: _proposal.totalAmount,
                 ),
-                // if (_proposal.status.toLowerCase() ==
-                //     "voting period")
-                VotingButtons(
-                  proposal: _proposal,
-                ),
+                if (_proposal.status.toLowerCase() == "voting period")
```

Comment:
Would it make sense for `"voting period"` to either be an enum or defined as a `const` with other statuses?

Reply:
Yes, it would.